recipes/occt/all/conanfile.py

```diff
@@ -0,0 +1,202 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+
+
+class OcctConan(ConanFile):
+    name = "occt"
```

Comment:
this lib is usually called opencascade in package managers: https://repology.org/project/opencascade/versions